scripts/migrate.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+ * SonarQube Apple Plugin - Enables analysis of Swift and Objective-C projects into SonarQube.
+ * Copyright © 2022 inside|app (contact@insideapp.fr)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+// FIXME : this is a temporary script to merge Sqale data with rules JSON file
+// Should be deleted when all linters will be migrated
+
+import groovy.json.JsonSlurper
+import groovy.json.JsonBuilder
+
+//print "What is your name? "
```

Comment:
Example code to remove?

Reply:
Removed !
Anyway this script is temporary, and will be removed once OCLintSensor is done.